### PR TITLE
Changes to make boolean type options full screen width while horizont…

### LINKED ISSUE #1968: [SDC] Change the alignment to make boolean type options full screen width
**Problem**
Boolean questions are not full screen width

**Solution**
Change the alignment to make boolean type options full screen width

**EmCare commit**
https://github.com/google/android-fhir/commit/f1408650e8ff71ee52169c1825e58249cdfbe784

cc @santosh-pingle @kunjan8794 @khyativyasargus @jingtang10

### PR DESCRIPTION
**Pull request by khyativyasargus**
…ally aligned

**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #1968 

**Description**
Change the alignment to make boolean type options full screen width while horizontally aligned

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

**Type**
Choose one: Feature

**Screenshots (if applicable)**
Screenshot_20230424_183545

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://github.com/google/android-fhir/wiki/Contributing) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [x] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

**Comment by santosh-pingle** [comment]
@khyativyasargus 
Please also add screenshot(with issue) before making this change just for reference.

**Comment by khyativyasargus** [comment]
@santosh-pingle Sure.

image

File: datacapture/src/main/java/com/google/android/fhir/datacapture/views/factories/BooleanChoiceViewHolderFactory.kt [hunk 1]
```diff
@@ -133,12 +134,13 @@ internal object BooleanChoiceViewHolderFactory :
         choiceOrientation: ChoiceOrientationTypes
       ) {
         layoutParams =
-          ViewGroup.LayoutParams(
+          LinearLayout.LayoutParams(
             when (choiceOrientation) {
-              ChoiceOrientationTypes.HORIZONTAL -> ViewGroup.LayoutParams.WRAP_CONTENT
+              ChoiceOrientationTypes.HORIZONTAL -> 0
```
Comment (santosh-pingle) [inline datacapture/src/main/java/com/google/android/fhir/datacapture/views/factories/BooleanChoiceViewHolderFactory.kt:139]:
```suggestion
              ChoiceOrientationTypes.HORIZONTAL -> /* width= */ 0
```

File: datacapture/src/main/java/com/google/android/fhir/datacapture/views/factories/BooleanChoiceViewHolderFactory.kt [hunk 2]
```diff
@@ -133,12 +134,13 @@ internal object BooleanChoiceViewHolderFactory :
...
               ChoiceOrientationTypes.VERTICAL -> ViewGroup.LayoutParams.MATCH_PARENT
             },
-            ViewGroup.LayoutParams.WRAP_CONTENT
+            ViewGroup.LayoutParams.WRAP_CONTENT,
+            1.0f
```
Comment (santosh-pingle) [inline datacapture/src/main/java/com/google/android/fhir/datacapture/views/factories/BooleanChoiceViewHolderFactory.kt:143]:
```suggestion
            /* weight= */ 1.0f
```

**Comment by santosh-pingle** [comment]
Thanks!
Added few optional comments.
LGTM

**Comment by khyativyasargus** [comment]
@santosh-pingle  
Here is the screenshot(with issue)

image

**Review by santosh-pingle — APPROVED** [review]
thanks! @khyativyasargus

**Pull request merged**